BitwardenShared/UI/Vault/Vault/VaultList/VaultListItem.swift

```diff
@@ -5,7 +5,6 @@ import Foundation
 ///
 public struct VaultListItem: Equatable, Identifiable, Sendable, VaultItemWithDecorativeIcon {
     // MARK: Types
-
```

Comment:
⛏️ There should be a space between a Mark and the next lines or a warning will appear. Please add the line back.